packages/aws-cdk-lib/aws-eks/lib/alb-controller.ts

```diff
@@ -90,7 +95,7 @@ export class AlbControllerVersion {
   /**
    * v2.4.2
    */
-  public static readonly V2_4_2 = new AlbControllerVersion('v2.4.2', '1.4.3', false);
+  public static readonly V2_4_2 = new AlbControllerVersion('v2.4.2', '1.4.2', false);
```

Comment:
Not all chart version would aligned with App version, for the version `v2.4.2` the chart version should be `1.4.3`.

You may check version mapping with the command below
```sh
helm repo update
helm search repo aws-load-balancer-controller --versions
```

Reply:
Fixed 👍